game/scripts/vscripts/GameMode.ts

```diff
@@ -272,6 +272,11 @@ export class GameMode {
                     }
                 });
             }
+
+            // Neutral creeps (Check for neutral team too not remove Slacks etc)
+            if (unit.IsNeutralUnitType() && unit.GetTeam() == DotaTeam.NEUTRALS && this.tutorial.currentSection && this.tutorial.currentSection.name !== SectionName.Chapter3_Opening) {
```

Comment:
Shouldn't hardcode section names here. We can add it to required state / setupState instead so on each section we make sure neutrals are / arent spawned.